src/api/ParticipantManager.API/Models/Encounter.cs

```diff
@@ -5,5 +5,5 @@ public class Encounter
   public Guid EncounterId { get; set; }
   public Guid EpisodeId { get; set; }
   public DateTime Date { get; set; }
-  public string Outcome { get; set; }
+  public string? Outcome { get; set; }
```

Comment:
Not sure about this one. 
I know that initially we are ingesting "Closed" encounters so I assume those will all have an Outcome.
Suggest making it required for now, easier to loosen it and make it nullable later if necessary